PreFreeSurfer/scripts/ACPCAlignment.sh

```diff
@@ -83,38 +104,87 @@ echo "date: `date`" >> $WD/log.txt
+  verbose_echo " --> Run EXVIVO brain registration using ReferenceBrain"
+  ${FSLDIR}/bin/flirt -interp spline -in "$WD"/robustroi.nii.gz -ref "$WD"/ReferenceBrain -omat "$WD"/roi2std.mat -out "$WD"/acpc_final.nii.gz -searchrx -30 30 -searchry -30 30 -searchrz -30 30
+elif [[ "$SPECIES" != "Human" ]] && [ $BrainExtract = INVIVO ] ; then
+  isopixdim=$(fslval "$Reference2mm" pixdim1)
+  #flirt -in "$WD"/robustroi.nii.gz -ref "$WD"/robustroi.nii.gz -applyisoxfm $isopixdim -o "$WD"/robustroi2mm.nii.gz -interp sinc
+  dim1=$(fslval "$WD"/robustroi.nii.gz dim1)
+  dim2=$(fslval "$WD"/robustroi.nii.gz dim2)
+  dim3=$(fslval "$WD"/robustroi.nii.gz dim3)
+  pixdim3=$(fslval "$WD"/robustroi.nii.gz pixdim3)
+  centerx=$(echo "$dim1*0.5" | bc | awk '{printf "%d", $1}')
+  centery=$(echo "$dim2*0.48" | bc| awk '{printf "%d", $1}')
+  centerz=$(echo "$dim3 - $BetTop2Center/$pixdim3" | bc | awk '{printf "%d", $1}') 
+  if [ "$BiasfieldCor" = TRUE ] ; then
+    BC="-B"
+  fi
+  verbose_echo " --> Run initial BET with options: -m -r $BetRadius -c $centerx $centery $centerz -f $BetFraction -z $betspecieslabel $BC"
+  ${HCPPIPEDIR_Global}/bet4animal "$WD"/robustroi.nii.gz "$WD"/robustroi_brain -m -r $BetRadius -c $centerx $centery $centerz -f $BetFraction -z $betspecieslabel $BC
+  verbose_echo " --> Registering brain extracted image to MNI152 (12 DOF)"
+  ${FSLDIR}/bin/flirt -interp spline -in "$WD"/robustroi_brain.nii.gz -ref "$WD"/ReferenceBrain -omat "$WD"/roi2std_init.mat -out "$WD"/acpc_final_init.nii.gz -searchrx -30 30 -searchry -30 30 -searchrz -30 30 -dof 6
+  verbose_echo " --> Registering cropped image to MNI152 (12 DOF)"
+  ${FSLDIR}/bin/flirt -interp spline -in "$WD"/robustroi.nii.gz -ref "$WD"/Reference -init "$WD"/roi2std_init.mat -omat "$WD"/roi2std.mat -out "$WD"/acpc_final.nii.gz -nosearch -dof 12 # -inweight "$WD"/robustroi_brain_mask -refweight "$Reference2mmMask" - 0609 did not work with these inweight & refweight
+else              # NONE 
+  verbose_echo " --> Registering cropped image to MNI152 (12 DOF)"
+  ${FSLDIR}/bin/flirt -interp spline -in "$WD"/robustroi.nii.gz -ref "$Reference" -omat "$WD"/roi2std.mat -out "$WD"/acpc_final.nii.gz -searchrx -30 30 -searchry -30 30 -searchrz -30 30
+fi
+
+verbose_echo " --> Getting a 6 DOF approximation"
 # Concatenate matrices to get full FOV to MNI
 verbose_echo " --> Concatenating matrices to get full FOV to MNI"
 ${FSLDIR}/bin/convert_xfm -omat "$WD"/full2std.mat -concat "$WD"/roi2std.mat "$WD"/full2roi.mat
 
 # Get a 6 DOF approximation which does the ACPC alignment (AC, ACPC line, and hemispheric plane)
 verbose_echo " --> Geting a 6 DOF approximation"
-$CARET7DIR/wb_command -convert-affine -from-flirt "$WD"/full2std.mat "$Input".nii.gz "$Reference" -to-world "$WD"/full2std_world.mat
+#${FSLDIR}/bin/aff2rigid "$WD"/full2std.mat "$OutputMatrix"
+${CARET7DIR}/wb_command -convert-affine -from-flirt "$WD"/full2std.mat "$Input".nii.gz "$Reference".nii.gz -to-world "$WD"/full2std_world.mat
```

Comment:
Change all of these back to not add a `.nii.gz` onto `"$Reference"`, since we are passing these as full filenames again.